### PR TITLE
Update to golang 1.20

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

**Comment by skorati** [comment]
:thumbsup: Coverage increased from [%] to [69.4%]

**Comment by skorati** [comment]
:thumbsup: Coverage increased from [%] to [69.3%]

**Comment by skorati** [comment]
:thumbsup: Coverage increased from [%] to [69.4875%]

File: Dockerfile.builder [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19
+FROM golang:1.20
```
Comment (ambiknai) [inline Dockerfile.builder:1]: arent we moving to `1.20.11` ??

Comment (sameshai) [reply Dockerfile.builder]: I was using generic release following old pattern. If we want specific version updating.

**Review by ambiknai — APPROVED** [review]

**Comment by skorati** [comment]
:thumbsup: Coverage increased from [%] to [69.1125%]

**Review by skorati — APPROVED** [review]
looks good.

**Pull request merged**